smarts/core/smarts.py

```diff
@@ -77,26 +78,41 @@ class SMARTSNotSetupError(Exception):
 
 
 class SMARTS:
+    """The SMARTS simulator. This is the direct interface to all parts of the simulation.
+    Args:
+        agent_interfaces: The interfaces providing SMARTS with the understanding of what features an agent needs.
+        traffic_sim: The traffic simulator for providing non-agent traffic.
+        envision: An envision client for connecting to an envision visualization server.
+        visdom: A visdom client for connecting to a visdom visualization server.
+        fixed_timestep_sec: The fixed timestep that will be default if time is not otherwise specified at step.
+        reset_agents_only: When specified the simulation will continue use of the current scenario.
+        zoo_addrs: The (ip:port) values of remote agent workers for externally hosted agents.
+        external_provider: Creates a special provider `SMARTS.external_provider` that allows for inserting state.
+        config: The simulation configuration file for unexposed configuration.
+    """
+
     def __init__(
         self,
-        agent_interfaces,
+        agent_interfaces: Dict[str, AgentInterface],
         traffic_sim,  # SumoTrafficSimulation
         envision: EnvisionClient = None,
         visdom: VisdomClient = None,
         fixed_timestep_sec: float = 0.1,
         reset_agents_only: bool = False,
-        zoo_addrs=None,
+        zoo_addrs: Optional[Tuple[str, int]] = None,
         external_provider: bool = False,
+        config=SmartsConfig(),
```

Comment:
I'm not sure about adding a custom `SmartsConfig` object just for renderer configuration, especially if there are (currently) only two values that might get set there.  I'm wondering if this can be done a different way.

Reply:
I removed the config and have a direct configuration to the renderer.